CHANGELOG.md

```diff
@@ -127,11 +127,20 @@ Additions to existing modules
   nonZeroIndex : Fin n → ℕ.NonZero n
   ```
 
-* In `Data.Integer.Divisisbility`: introduce `divides` as an explicit pattern synonym
+* In `Data.Integer.Divisibility`: introduce `divides` as an explicit pattern synonym
   ```agda
   pattern divides k eq = Data.Nat.Divisibility.divides k eq
   ```
 
+* In `Data.Integer.Properties`:
+  ```agda
+  ◃-nonZero : {{_ : ℕ.NonZero n}} → NonZero (s ◃ n)
+  sign-*    : {{NonZero (i * j)}} → sign (i * j) ≡ sign i Sign.* sign j
```

Comment:
Missing irrelevance annotation on the two lines above?

Reply:
Done